client/components/domains/registrant-extra-info/fr-form.jsx

```diff
@@ -0,0 +1,273 @@
+
+const debug = debugFactory( 'calypso:domains:registrant-extra-info' );
+
+const countriesList = countriesListForDomainRegistrations();
+
+class RegistrantExtraInfoForm extends React.PureComponent {
+	static propTypes = {
+		isVisible: PropTypes.bool,
+		onSubmit: PropTypes.func,
+		countriesList: PropTypes.object,
+		onStateChanged: PropTypes.func, // Just until we can reduxify the contact details
+	}
+
+	static defaultProps = {
+		isVisible: true,
+		isProbablyOrganization: false,
+		onSubmit: noop,
+		countriesList: countriesList,
+	}
+
+	constructor( props ) {
+		super( props );
+	}
+
+	componentWillMount() {
+		this.setState( {
+			registrantType: this.props.isProbablyOrganization
+				? 'organization' : 'individual',
+			countryOfBirth: 'AU',
+			dateOfBirth: '1978-01-01',
+			placeOfBirth: 'dummyCity',
+			postalCodeOfBirth: '12345',
+			registrantVatId: 'XX123456789',
+			sirenSiret: '123456789',
+			trademarkNumber: '123456789',
+		} );
+	}
+
+	getRelevantFields( state ) {
+		const { countryOfBirth, registrantType } = state;
+		const bornInFrance = countryOfBirth === 'FR';
+		const birthPlaceFields = [ 'placeOfBirth', 'postalCodeOfBirth' ];
+		const individualFields = [
+			'countryOfBirth',
+			'dateOfBirth',
+			...( bornInFrance ? birthPlaceFields : [] ),
+		];
+		const organizationalFields = [ 'registrantVatId', 'sirenSiret', 'trademarkNumber' ];
+		const conditionalFields = registrantType === 'individual'
+			? individualFields
+			: organizationalFields;
+		return [ 'registrantType', ...conditionalFields ];
+	}
+
+	handleDobChangeEvent = ( event ) => {
+		this.handleChangeEvent( event );
+		// setState() is not syncronous :(
+		const updatedState = {
+			...this.state,
+			[ event.target.name ]: event.target.value,
+		};
+		const { dob_years: years, dob_months: months, dob_days: days } = updatedState;
+
+		if ( years && months && days ) {
+			const value = [ years, months, days ].join( '-' );
+			const pseudoEvent = {
+				target: {
+					name: 'dateOfBirth',
+					value
+				}
+			};
+			debug( 'Setting dateOfBirth to ' + value +
+				( moment( value, 'YYYY-MM-DD' ).isValid() ? '' : ' (invalid)' ) );
+			this.handleChangeEvent( pseudoEvent );
+		}
+	}
+
+	handleChangeEvent = ( event ) => {
+		this.setState( { [ event.target.name ]: event.target.value } );
+	}
+
+	componentWillUpdate( _ /* nextProps */, nextState ) {
+		// This is pretty dirty :(
+		// The sooner we can get the contact details into the state, the better
+		this.props.onStateChanged && this.props.onStateChanged( nextState );
+	}
+
+	render = () => {
+		return (
+			<form>
+				<FormFieldset>
+					<FormLegend>
+						{ this.props.translate( "Who's this domain for?" ) }
+					</FormLegend>
+					<FormLabel>
+						<FormRadio value="individual"
+							name="registrantType"
+							checked={ 'individual' === this.state.registrantType }
+							onChange={ this.handleChangeEvent } />
+						<span>{ this.props.translate( 'An individual' ) }</span>
```

Comment:
Hi! I've found a possible matching string that has already been translated **7** times:
`translate( 'Individual' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).